examples/with-supabase/components/ui/badge.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from "react";
```

Comment:
All shadcn files are in `components/ui` but they should be in `components` folder according to the `components.json`. Change `components.json` (probably more correct) or move them back.

Reply:
shadcn adds `ui` directory itself

Reply:
its just not well documented

Reply:
this components.json file is what placed them in the right directory, i used the CLI to add components